app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -496,17 +496,17 @@ function findConditionalsRegexErrors(content) {
   }
 }
 
-function findConditionalsElseIfUses(content) {
-  const regex = /@else[\s\n]+@?if/g
+function findConditionalsElifUses(content) {
```

Comment:
Let's use the full name for this function

Reply:
Psst: I think this is on purpose. It is finding literally "elif", not "else if".

Reply:
Oh of course this is the linter, I got it confused with the compiler